src/hotspot/share/gc/shared/gcConfiguration.cpp

```diff
@@ -72,12 +72,7 @@ GCName GCConfiguration::old_collector() const {
   }
 
   if (UseShenandoahGC) {
-#if INCLUDE_SHENANDOAHGC
-    if (strcmp(ShenandoahGCMode, "generational") == 0) {
-      return Shenandoah;
-    }
-#endif
-    return NA;
+    return Shenandoah;
```

Comment:
@earthling-amzn : Here and at lines 47-52, don't we want to return, much like Gen ZGC does, the (new) values `ShenandoahMinor` and `ShenandoahMajor` -- or ShYoung and ShOld, or ShNew and ShOld, as one prefers -- for the young and old collectors, respectively, in the generational case, and just Shenandoah for both cases in the non-generational case?

Where are the `GCConfiguration::*_collector()` APIs called from?

Reply:
We could do that on a different PR.